diff --git a/src/test/scala/svc/domain/MoneySpec.scala b/src/test/scala/svc/domain/MoneySpec.scala
index 925ba6f..8a27ba3 100644
--- a/src/test/scala/svc/domain/MoneySpec.scala
+++ b/src/test/scala/svc/domain/MoneySpec.scala
@@ -3,9 +3,6 @@ package svc.domain
 import org.scalatest.funsuite.AnyFunSuite
 
 class MoneySpec extends AnyFunSuite:
-  test("adds") {
-    assert(Money.add(Money(100, "EUR"), Money(200, "EUR")) == Right(Money(300, "EUR")))
-  }
 
   test("refuses mixed currencies") {
     assert(Money.add(Money(1, "EUR"), Money(1, "USD")).isLeft)
