diff --git a/test/svc/domain/money_test.clj b/test/svc/domain/money_test.clj
index 2a18b8a..19078aa 100644
--- a/test/svc/domain/money_test.clj
+++ b/test/svc/domain/money_test.clj
@@ -5,8 +5,7 @@
             [clojure.test.check.properties :as prop]
             [svc.domain.money :as money]))
 
-(deftest adds
-  (is (= (money/money 5 "EUR") (money/add (money/money 2 "EUR") (money/money 3 "EUR")))))
+
 
 (deftest refuses-mixed-currencies
   (is (= {:error :mixed-currencies} (money/add (money/money 2 "EUR") (money/money 3 "USD")))))
